### ISSUE TITLE
Pontoon-wagtail stopped syncing

### ISSUE THREAD

**Issue by TheoChevalier** [mozilla/donate-wagtail#520]
Something in this commit seem to have broken pontoon-wagtail sync on staging: https://github.com/kaedroho/mozilla-donate-staging-content/commit/76f1b5c59723b3f7d0a0e0fa0e8596674bb8ff9f

Now when I hit `Force sync`, it doesn’t pull the translations above, nor does it push new source changes.

My suspicion is on the straight quote `'` in the last string. We use curly quotes `’` in French, but many locales don’t, so I wanted to test that. But I could be wrong and it could be something else.

**Comment by TheoChevalier** [comment]
Removing the straight quote here didn’t help https://github.com/kaedroho/mozilla-donate-staging-content/commit/ce138891abb8c8d7c2ff3ef6b9b59963dcbef8f1

So either it’s stuck and not recovering, or this wasn’t the issue in the first place

**Comment by alanmoo** [comment]
What do we need to do to address this?

**Comment by patjouk** [comment]
It's throwing a python error, I'll take a look now.

```
2019-11-01T14:58:53.881242+00:00 app[worker.1]: > /app/.heroku/python/lib/python3.7/site-packages/wagtail_localize_pontoon/models.py(173)find_translatable_submission()
2019-11-01T14:58:53.881256+00:00 app[worker.1]: -> for submission in submissions_to_check:
2019-11-01T14:58:53.881414+00:00 app[worker.1]: (Pdb)
2019-11-01T14:58:53.917176+00:00 app[worker.1]: 2019-11-01 14:58:53,916 [ERROR] bdb.BdbQuit
```

**Comment by cadecairos** [comment]
https://sentry.prod.mozaws.net/operations/donate-prod/issues/6622305/

public link: https://sentry.prod.mozaws.net/share/issue/23f35fd79ecf4d4d98994bc23e72e3e0/

```
BdbQuit: 
(9 additional frame(s) were not displayed)
...
  File "wagtail_localize_pontoon/importer.py", line 117, in try_update_resource_translation
    translatable_submission = resource.find_translatable_submission(language)
  File "wagtail_localize_pontoon/models.py", line 173, in find_translatable_submission
    for submission in submissions_to_check:
  File "wagtail_localize_pontoon/models.py", line 173, in find_translatable_submission
    for submission in submissions_to_check:
  File "bdb.py", line 88, in trace_dispatch
    return self.dispatch_line(frame)
  File "bdb.py", line 113, in dispatch_line
    if self.quitting: raise BdbQuit
```

**Comment by patjouk** [comment]
Removing the scheduled task that sync with Pontoon  for now.

**Comment by cadecairos** [comment]
@kaedroho Do you know what would cause the wagtail-localize-pontoon module to raise this exception?

**Comment by TheoChevalier** [comment]
I wonder if it could be as simple as that https://stackoverflow.com/a/48719897

Coming from https://github.com/kaedroho/wagtail-localize-pontoon/blob/50397b0103055cac36887a80120a6d1d04660ed8/wagtail_localize_pontoon/models.py#L183

**Comment by kaedroho** [comment]
Oops! Sorry I'm not sure how I missed that in my testing, I'll remove it now!

**Comment by kaedroho** [comment]
I've just released version 0.1.1 which has this statement removed.

**Comment by TheoChevalier** [comment]
Thanks Karl!
